### PR TITLE
remove arm64+updatetoOneDotOh

### PR DESCRIPTION
**Pull request by kelseiv**

### CODE REVIEW

File: recipes/pulsar/docker-compose.yml [hunk 1]
```diff
@@ -54,7 +54,7 @@ services:
     depends_on:
       - pinot-broker
     volumes:
-      - ./plugins/pinot-pulsar-0.12.0-shaded.jar:/opt/pinot/plugins/pinot-stream-ingestion/pinot-pulsar/pinot-pulsar-0.12.0-arm64-shaded.jar
+      - ./plugins/pinot-pulsar-0.12.0-shaded.jar:/opt/pinot/plugins/pinot-stream-ingestion/pinot-pulsar/pinot-pulsar-0.12.0-arm64-shaded.jar
```
Comment (kelseiv) [inline recipes/pulsar/docker-compose.yml:57]: @mayankshriv @xiangfu0 is it accurate to update the pinot-pulsar jar to 1.0.0?

Comment (xiangfu0) [reply recipes/pulsar/docker-compose.yml:57]: we can use 1.0.0
put a note that apple is using `arm64`.
Intel` servers are using `amd64`

**Review by xiangfu0 — APPROVED** [review]

File: recipes/pulsar/docker-compose.yml (hunk 1 above)
Comment (kelseiv) [reply recipes/pulsar/docker-compose.yml:57]: To be sure, you're only referring to this pulsar instance? Noticed when installing pinot on Apple -arm64 tag no longer works... only 1.0.0 by itself. Is this your understanding too? Thank you, @xiangfu0!

**Review by matthewhelmke — APPROVED** [review]
LGTM, thank you! I'm going to merge this now so it's in there before we edit stuff.

**Pull request merged**